### PR TITLE
[1LP][RFR] Add remaining manual tests for Compare Host/Node view to test_infra_manual.py

### PR DESCRIPTION
**Pull request by prichard77**
## Purpose or Intent
- __Add manual tests__ for FA infra_hosts to cover remaining cases for the hosts comparison view.

### CODE REVIEW

File: cfme/tests/infrastructure/test_infra_manual.py [hunk 1]
```diff
@@ -288,39 +288,234 @@ def test_host_comparison_my_company_tags():
...
+                    7. Click on a child item, to deselect, and click "Apply".
+                    8. Click on random child items within the section and click "Apply" until no
+                       child items remain selected.
+                    9. Click on random child items within the section and click "Apply" until all
+                       child items remain selected.
+                    10. Repeat steps 6 through 9 for Configuration, Security, and Properties
+                        sections.
+
+                expectedResults:icon
+                    1. Hosts view is displayed.
```
Comment (ganeshhubale) [inline cfme/tests/infrastructure/test_infra_manual.py]: Typo: ```expectedResults:icon```
I think, you can remove that extra ```icon```. Also you can remove block line(504)

Comment (prichard77) [reply cfme/tests/infrastructure/test_infra_manual.py]: Updated as suggested.

File: cfme/tests/infrastructure/test_infra_manual.py [hunk 2]
```diff
@@ -288,39 +288,232 @@ def test_host_comparison_my_company_tags():
...
+            6. The "% Matched" and graphs displayed on child rows will be replaced with "+" or
+                  "-" depending if the item exists in the non-base hosts.
+    """
+    pass
+
+
+@test_requirements.infra_hosts
+def test_host_comparison_compressed_view():
+    """
+        Polarion:
```
Comment (john-dupuy) [inline cfme/tests/infrastructure/test_infra_manual.py]: Required: this is indented one block too far

File: cfme/tests/infrastructure/test_infra_manual.py [hunk 3]
```diff
@@ -288,39 +288,232 @@ def test_host_comparison_my_company_tags():
...
+                   - on child(leaf) rows, color coded values are replaced with checkmarks for
+                     matching, and x's for non-matching values
+    """
+    pass
+
+
+@test_requirements.infra_hosts
+def test_host_comparison_report():
+    """
+        Polarion:
```
Comment (john-dupuy) [inline cfme/tests/infrastructure/test_infra_manual.py]: Same issue here

File: cfme/tests/infrastructure/test_infra_manual.py (hunk 2 above)
Comment (prichard77) [reply cfme/tests/infrastructure/test_infra_manual.py]: updated as suggested

File: cfme/tests/infrastructure/test_infra_manual.py (hunk 3 above)
Comment (prichard77) [reply cfme/tests/infrastructure/test_infra_manual.py]: updated as suggested

File: cfme/tests/infrastructure/test_infra_manual.py [hunk 4]
```diff
@@ -288,39 +288,232 @@ def test_host_comparison_my_company_tags():
...
+
+
+@test_requirements.infra_hosts
+def test_host_comparison_multipleviews_interactions():
+    """
+    Here i will test adding and removing rows and sections in permutations I did not cover
+    explictily already. Also cover the case where clicking on the parent section selects all of
+    the child sections.
+
+        Polarion:
```
Comment (john-dupuy) [inline cfme/tests/infrastructure/test_infra_manual.py]: One last indent issue

**Pull request merged**